src/main/java/mate/academy/service/MovieServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.service;
+
+import exception.MovieNotFoundException;
+import mate.academy.dao.MovieDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Movie;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    @Inject
+    private MovieDao movieDao;
+
+    @Override
+    public Movie add(Movie movie) {
+        return movieDao.add(movie);
+    }
+
+    @Override
+    public Movie get(Long id) {
+        return movieDao.get(id).orElseThrow(() ->
+                new MovieNotFoundException("Not found movie with id = " + id + " from DB")
```

Comment:
let's just throw runtime exception here, there is no need to create a custom exception for every case